### DISCUSSION TITLE
Count posts and boosts separately

### DISCUSSION THREAD

**Discussion by StyXman** [cheeaun/phanpy#351] [Ideas]
See this:

https://phanpy.social/#/en.osm.town/a/107081404834310301

If you remove boosts, the posts count goes from 115 to 0.

**Comment by cheeaun** [comment]
For "the posts count goes from 115 to 0", which UI/part are you referring to?

**Reply by eriol** [reply]
Maybe the list of posts? I mean:

![image](https://github.com/cheeaun/phanpy/assets/212279/a25e842b-f184-464b-9fd9-2c213dedf32a)

**Reply by cheeaun** [reply]
If the API returns nothing, the account's timeline probably only have boosts.

**Reply by StyXman** [reply]
yeah, that's why I suggest counting posts and boosts separately.

**Reply by cheeaun** [reply]
@StyXman are you saying that this UI that shows the count here should show 0 if "remove boosts" filter is applied?
![Screenshot 2023-12-17 at 7 55 23 PM](https://github.com/cheeaun/phanpy/assets/2296/8e8acb9e-fb2b-463f-8f32-1b241174f202)

Or you mean the profile should show two counts like "0 Posts 115 Boosts"?
